app/routes/cart/route.tsx

```diff
@@ -90,7 +90,9 @@ export default function CartPage() {
                         </span>
                     </div>
                     <div className={styles.summaryRow}>
-                        <span className={styles.location}>Poland</span>
+                        <span className={styles.shippingInfo}>
+                            {cartTotals?.shippingInfo?.region?.name}
```

Comment:
I'm not sure if it could be undefined, but don't display `summaryRow` when it is absent.

Reply:
I looks ok without data, but I've fixed it to get rid of redundant padding.